src/pysource/sensitivity.py

```diff
@@ -62,6 +64,9 @@ def crosscorr_time(u, v, model, **kwargs):
     model: Model
         Model structure
     """
+    if model.is_viscoacoustic:
```

Comment:
I still think this needs the time spacing (`w` below) to be correct

Reply:
As we discussed above...when we put `w`, the test fails.